packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -189,6 +189,10 @@ contract L2Genesis is Deployer {
         if (writeForkGenesisAllocs(_fork, Fork.ISTHMUS, _mode)) {
             return;
         }
+
+        if (writeForkGenesisAllocs(_fork, Fork.JOVIAN, _mode)) {
```

Comment:
Does this need an `activateJovian` call before the `if` block, similar to the pattern in many of the forks above (e.g. `activateIsthmus`)? I'm not familiar with what's in Jovian, so maybe this is not necessary